vscode-dotnet-runtime-library/src/Utils/IssueReporter.ts

```diff
@@ -7,7 +7,7 @@ import { IIssueContext } from './IIssueContext';
 // tslint:disable no-var-requires
 const packageJson = require('../../package.json');
 
-const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues`;
+const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues/new`;

```

Comment:
```suggestion
const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues/new/choose`;
```

I think this should end in choose so it will show all of the templates, but I could be wrong ;)

Reply:
Yes you are right! Thank you so much!